### PR TITLE
Allow specific searches

### PR DESCRIPTION
**Pull request by seddy**

### CODE REVIEW

**Comment by sadir** [comment]
Is this ready for review then given your comment? It looks good overall. What issue are you seeing with config in other apps?

**Comment by seddy** [comment]
So essentially I've set the `mock_client_fixtures_dir` in the config here, but that now overwrites the directory set in `elixir-umbrella/core`, so when I test `elixir-umbrella` using this version of the package, anything that uses relies on custom fixtures blows up :sob:

**Comment by seddy** [comment]
To answer your query; this is probably fine for review, I just need to work out how to get around the config issue (which may or may not require some tweaks here)

**Comment by sadir** [comment]
Fair enough - I'm sure there is a way to only set it if it is not set already. We do that in our umbrella already in places.

File: mix.exs [hunk 1]
```diff
@@ -26,7 +26,8 @@ defmodule Closex.Mixfile do
   end
 
   def application do
-    [mod: {Closex.Application, []}]
+    [mod: {Closex.Application, []},
+     env: [mock_client_fixtures_dir: "test/fixtures"]]
```
Comment (samphilipd) [inline mix.exs]: Why is this set here?

Comment (seddy) [reply mix.exs]: Ignore this, I'm trying to debug an issue when using this change

**Pull request merged**